### PR TITLE
DAP-1005: Format Date as YYYY-MM-DD

### PR DESCRIPTION
**Pull request by BradyMitch**
## 🎯 Summary

  
[DAP-1005](https://citz-cirmo.atlassian.net/browse/DAP-1005)

- Modified formatDate in backend
- Modified date format in frontend

## 🔰 Checklist

- [x] I have read and agree with the following checklist.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

**Comment by BradyMitch** [comment]
Waiting on merge of DAP-1003

**Review by TaylorFries — COMMENTED** [review]
The default format of rows within file-list FolderDisplayGrid are "MM/DD/YYYY" is there any way of changing this?

**Comment by TaylorFries** [comment]
sorry just noticed that this is still a draft

File: backend/src/modules/filelist/utils/queueConsumer.ts [hunk 1]
```diff
@@ -39,10 +40,12 @@ export const queueConsumer = async (msg: amqp.ConsumeMessage, channel: amqp.Chan
 		return console.error("Email not found in filelist.");
 	}
 
+	const date = formatDate(new Date().toISOString());
```
[the remark this answers is not in the document]
Comment (BradyMitch) [reply backend/src/modules/filelist/utils/queueConsumer.ts:43]: It formatted date should be YYYY-MM-DD in the file name

**Comment by BradyMitch** [comment]
> The default format of rows within file-list FolderDisplayGrid are "MM/DD/YYYY" is there any way of changing this?

The format was already set for when in view mode but yes it turns out it is possible to set in edit mode as well so Ive just added that.

**Review by TaylorFries — APPROVED** [review]
dates match format as requested by SME

**Pull request merged**